components/NavButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useTranslations } from 'next-intl'
+
+type Props = {
+  activeQuestion: number
+  connected: boolean
+  lastQuestion: boolean
+  nextQuestion: () => void
+  prevQuestion: () => void
+  surveyOver: boolean
+}
+
+const NavButtons = ({
+  activeQuestion,
+  connected,
+  lastQuestion,
+  nextQuestion,
+  prevQuestion,
+  surveyOver
```

Comment:
`activeQuestion`, `connected`, and `surveyOver` are only used to disable the buttons so it might make more sense to make a new prop `nextDisabled` `prevDisabled` and then do these checks outside the component? Or you could get really fancy and say if `nextQuestion` is null the button gets disabled but I'd document that carefully

Reply:
Ooh good idea!!